src/components/homepage/JoinCTA.astro

```diff
@@ -0,0 +1,23 @@
+<section class="w-full flex-col justify-start items-center bg-gray-700 text-white py-8">
```

Comment:
We are using bg-gray-800 as secondary background and bg-gray-900 as primary. I think doing so will add uniformity through out the site.

We can have a separate issue for configuring brand and coloring through some configurables.

Reply:
making the background secondary will make it seem like it is joined to footer, which of this do you want?
increase height of border or its good?
![image](https://github.com/user-attachments/assets/8d7f20e0-abf8-4ef1-8684-751116ba3e48)
vs
![image](https://github.com/user-attachments/assets/3c2f0452-ce6e-493e-b522-67f2066144d5)